pkgs/build-support/fetchsteam/default.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  runCommand,
+  depotdownloader,
+  cacert,
+  writeText,
+}:
+
+{
+  name ? "${toString app}-${toString depot}-${toString manifest}-depot",
+  app,
+  depot,
+  manifest,
+  branch ? null,
+  language ? null,
+  lowViolence ? false,
+  fileList ? [ ],
+  fileListRegex ? false,
+  debug ? false,
+  hash ? lib.fakeHash,
```

```suggestion
  hash ? "",
```
This is more inline with fetchers like `fetchCargoTarball` and `pnpm.fetchDeps`

---

I don't think the added complexity of doing it that way is particularly worth it. Those fetchers probably predate `lib.fakeHash`, so they chose the way that was simpler back then.

If you want to block on this, I'll change it, but I'd strongly prefer to keep it this way.